UnityProject/Assets/Scripts/Core/Lifecycle/PlayerSpawn.cs

```diff
@@ -210,8 +212,13 @@ public static GameObject ServerClonePlayer(Mind forMind, Vector3Int worldPositio
 	private static Vector3Int GetSpawnPointForOccupation(Occupation occupation)
 	{
 		Transform spawnTransform;
+		if (occupation == null)
+		{
+			spawnTransform = SpawnPoint.GetRandomPointForJob(JobType.ASSISTANT);
+			return spawnTransform != null ? spawnTransform.position.CutToInt() : Vector3Int.zero;
+		}
 		//Spawn normal location for special jobs or if less than 2 minutes passed
-		if (GameManager.Instance.stationTime < ARRIVALS_SPAWN_TIME || occupation.LateSpawnIsArrivals == false)
```

Comment:
why remove the second condition?

Reply:
Whoops, forgot to add this back in while testing. re-added it back.